src/app/digital/public/api/impl/DigitalComponent.ts

```diff
@@ -1,4 +1,6 @@
+import { CircuitInternal } from "core/internal";
```

Comment:
nit: remove spaces in import, this should be handled by ESLint, definitely install the VSCode Eslint plugin if you haven't already